packages/grafana-data/src/valueFormats/dateTimeFormatters.test.ts

```diff
@@ -179,6 +180,26 @@ describe('duration', () => {
     const str = toDurationInHoursMinutesSeconds(0);
     expect(formattedValueToString(str)).toBe('00:00:00');
   });
+  it('1 dtdhms', () => {
+    const str = toDurationInHoursMinutesSeconds(1);
+    expect(formattedValueToString(str)).toBe('00:00:01');
+  });
+  it('-1 dtdhms', () => {
+    const str = toDurationInHoursMinutesSeconds(-1);
+    expect(formattedValueToString(str)).toBe('00:00:01 ago');
+  });
+  it('0 dtdhms', () => {
+    const str = toDurationInHoursMinutesSeconds(0);
+    expect(formattedValueToString(str)).toBe('00:00:00');
+  });
+  it('1179811 dtdhms', () => {
+    const str = toDurationInDaysHoursMinutesSeconds(1179811);
+    expect(formattedValueToString(str)).toBe('13 d 15:43:31');
+  });
+  it('-1179811 dtdhms', () => {
+    const str = toDurationInDaysHoursMinutesSeconds(-1179811);
+    expect(formattedValueToString(str)).toBe('13 d 15:43:31 ago');
+  });
```

Comment:
Maybe add a unit test that verifies the output when we will get single digit days.